### ISSUE TITLE
There seems to be a problem with the install_deps.sh

### ISSUE THREAD

**Issue by HERMIT-OuO** [p4lang/behavioral-model#1070]
`libssl1.0-dev` seems to exist only in Ubuntu 18.04, in Ubuntu 20.04 this dependency should be installed using `libssl-dev`.

However, there is the following code in `install_deps.sh` that causes 20.04 to still use `libssl1.0-dev`, so hopefully this can be fixed.

```shell
ubuntu_release=`lsb_release -s -r`
if [[ "${ubuntu_release}" > "18" ]]
then
    # This older package libssl1.0-dev enables compiling Thrift 0.9.2
    # on Ubuntu 18.04.  Package libssl-dev exists, but Thrift 0.9.2
    # fails to compile when it is installed.
    # TBD: whether using this package makes a difference for Ubuntu
    # 18.04 and Thrift 0.11.0.
    LIBSSL_DEV="libssl1.0-dev"
else
    LIBSSL_DEV="libssl-dev"
fi
```

**Comment by jafingerhut** [comment]
I have been using this patch in a script for installing open source P4 development tools on multiple versions of Ubuntu, including Ubuntu 20.04, for a year or so now.  I'm happy to create a PR for this repo if there is interest: https://github.com/jafingerhut/p4-guide/blob/master/bin/patches/behavioral-model-use-correct-libssl-pkg.patch

**Comment by HERMIT-OuO** [comment]
I've had no additional problems installing with `libssl-dev` in Ubuntu 20.04 for now, and I really can't find `libssl1.0-dev` in the 20.04 version of APT sources, so I think your change should be a good way to handle it.

**Comment by antoninbas** [comment]
Is someone going to open a PR to fix the script? We require Thrift >= 0.11 now so it doesn't make sense to have a workaround just to support older versions of Thrift.

**Comment by jafingerhut** [comment]
Opened PR here, after testing that my script for installing all open source P4 dev tools on Ubuntu 18.04 and 20.04 still works with these changes: https://github.com/p4lang/behavioral-model/pull/1073

(using Thrift 0.11.0 and other dependency versions currently documented in README)
